### ISSUE TITLE
Feature: code for Masakhane News and Bloom-lm datasets

### ISSUE THREAD

**Issue by cleong110** [krypticmouse/double-bind-training#8]
- [ ] Create snippets that take in a language code and download from Masakhane News/Bloom-lm
- [ ] Combine that data with train.txt, etc. from other datasets
- [ ] Make sure it all gets tagged in Weights and Biases!
- [ ] Incorporate into repo AND colab notebook for AfricaNLP

**Comment by manny-uncharted** [comment]
Is this still open, and should the snippets be in a separate file

**Comment by cleong110** [comment]
@manny-uncharted yes, still open as far as I know. The snippets can be in a separate file, sure. They will eventually be incorporated into one of the notebooks, since that's how we run our experiments for this project.

**Comment by manny-uncharted** [comment]
Okay thanks I'll get started on this then

**Comment by manny-uncharted** [comment]
@cleong110 do I make a new branch to make pull requests

**Comment by cleong110** [comment]
That seems like a good idea to me! The branch by @krypticmouse is essentially our master branch, we should merge that into master, then merge yours

**Comment by cleong110** [comment]
For now though, just branch off of his https://github.com/krypticmouse/double-bind-training/tree/train-lm-adapter and merge back into his as well.

**Comment by manny-uncharted** [comment]
@cleong110 apologies for delaying on this task, I've been quite engaged with school work, I've spent some time to study the bloom-lm dataset and the masakhane ner 2.0 dataset.

Here are a few things I noticed.
- The naming conventions differ for both dataset, so I'd have to work around creating a new naming convention, that allows me to capture the different naming conventions. 
- also how do I ensure it gets tagged in weights and biases

**Comment by krypticmouse** [comment]
@manny-uncharted if possible keep the naming conventions consistent with the MAFAND dataset. Also for the weights and bias things add the dataset names you're using as an environment variable then pass them in the tags parameter in the call. Refer to docs in README.

Let me know if you need any help.
